melodies_monet/plots/surfplots.py

```diff
@@ -21,6 +21,9 @@
 from monet.util.tools import get_epa_region_bounds as get_epa_bounds 
 import math
 from ..plots import savefig
+import datetime, pytz
```

Comment:
On line 10, since you don't use the calc_8hr_rolling_max, calc_24hr_ave from MONET anymore. Should we delete this line?

Reply:
If we do not need keep it as an option, or will not use them in the future, we can delete this line

Reply:
I would just delete this line. And also delete the commented lines where you call these functions since we are not using them anymore it makes it easier for people to understand how the code works.